features/term-management-by-slug.feature

```diff
@@ -0,0 +1,88 @@
+Feature: Manage WordPress terms by slug or ID
```

Comment:
Can we keep this in the existing features files? The feature filenames mostly correlate with their commands, which is a pattern we'll want to continue to follow.